e2e/content-services/version/version-actions.e2e.ts

```diff
@@ -147,10 +147,14 @@ describe('Version component actions', () => {
         versionManagePage.checkFileVersionExist('1.0');
     });
 
-    it('[C280007] Should be possible restore an old version of your file', () => {
+    it('[C280007] Should be possible to restore an old version of your file and the document list updated', async () => {
+        await browser.refresh();
```

Comment:
please remove this browser refresh is not needed, avoid to use the refresh it slow down the test.